flink-python/setup.py

```diff
@@ -316,16 +316,10 @@ def extracted_output_files(base_dir, file_path, output_directory):
                         'python_full_version >= "3.7" and platform_system != "Windows"',
                         'httplib2>=0.19.0,<=0.20.4', apache_flink_libraries_dependency]
 
-    if sys.version_info < (3, 7):
-        # python 3.6 upper and lower limit
-        install_requires.append('numpy>=1.14.3,<1.20')
-        install_requires.append('pandas>=1.0,<1.2.0')
-        install_requires.append('pyarrow>=0.15.1,<7.0.0')
-    else:
-        # python 3.7, 3.8 and 3.9 upper limit and M1 chip lower limit,
-        install_requires.append('numpy>=1.21.4,<1.22.0')
-        install_requires.append('pandas>=1.3.0,<1.4.0')
-        install_requires.append('pyarrow>=5.0.0,<9.0.0')
+    # python 3.7, 3.8 and 3.9 upper limit and M1 chip lower limit,
```

Comment:
since we drop the support python3.6, we can put these install dependencies in `install_requires` directly.